packages/icons-business-suite/README.md

```diff
@@ -28,6 +28,28 @@ Example usage with `<ui5-icon>` web component:
 <ui5-icon name="business-suite/1x2-grid-layout"></ui5-icon>
 ```
 
+The package provides two versions of each icon (Business Suite Icons verson 1.x and Business Suite Icons version 2.x).
```

Comment:
Spotted a typo in "verson".

Furthermore, let's use single naming to refer the collections
- Business Suite Icons version 1.x or SAP Icons Business suite V1
- Business Suite Icons version 2.x or SAP Icons Business suite V2

I suggest SAP Business Suite Icons  v1 and  SAP Business Suite Icons v2